### PR TITLE
Add server and config services for dashboards plugin

### PR DESCRIPTION
**Pull request by joshuali925**
Signed-off-by: Joshua Li <joshuali925@gmail.com>

### Description
Add server routes and services for get, create, update, delete configs

### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: dashboards-notifications/public/services/utils/helper.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import {
```
Comment (davidcui-amzn) [inline dashboards-notifications/public/services/utils/helper.ts:1]: missing copyright notice

**Review by davidcui-amzn — APPROVED** [review]
LGTM

File: dashboards-notifications/public/utils/constants.ts [hunk 2]
```diff
@@ -25,6 +25,8 @@
  */
 
 export const DOCUMENTATION_LINK = '';
```
Comment (zhongnansu) [inline dashboards-notifications/public/utils/constants.ts:27]: remove unused constant?

**Review by zhongnansu — APPROVED** [review]

File: dashboards-notifications/public/utils/constants.ts (hunk 2 above)
Comment (joshuali925) [reply dashboards-notifications/public/utils/constants.ts:27]: we don't have docs yet so it's empty, but the variable is used here 
https://github.com/opensearch-project/notifications/blob/develop/dashboards-notifications/public/pages/Notifications/components/EmptyState/EmptyState.tsx#L45

**Pull request merged**